docs/api/app.md

```diff
@@ -1231,7 +1233,9 @@ Returns `Object`:
   * `args` String[] (optional) _Windows_ - The command-line arguments to pass to
     the executable. Defaults to an empty array. Take care to wrap paths in
     quotes.
-
+  * `enabled` Boolean (optional) _Windows_ - `true` if the app registry key is startup approved.
```

Comment:
"if the app registry key is startup approved and therefore shows as enabled in Task Manager and Windows settings